.github/workflows/worker.yml

```diff
@@ -27,12 +34,26 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
 
       - uses: actions/setup-java@v4
         with:
           distribution: 'zulu'
           java-version: ${{ inputs.jdk }}
 
+      - name: setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 16.17.0
```

Comment:
I wonder if we could re-use what the project already uses?
could you try adding something like this to the script?
```
mvn  com.github.eirslett:frontend-maven-plugin:install-node-and-npm@install-node-and-npm -pl web-console/
PATH+=:web-console/target/node/
```

Reply:
Wow, interesting!
This worked! Thanks for the suggestion!